### DISCUSSION TITLE
Question Controller

### DISCUSSION THREAD

**Discussion by radthenone** [eadwinCode/django-ninja-extra#168] [General]
I have services and controllers. My Model1.service have method in controller to create with model.save() in repository model, but i use redis pub/sub to communicate between services. So Model1Service method create model in queryset but no create it in DB because method not ends and publisher send and wait for response from Model2Service and now i have problemvbecause handler in Model2Service listen publisher from Model1 but when i want create nested model this cant be create with ID from Model1 because Nothing exists in DB table. My question is controller with route.post have inside transactions ? can i off this or something? Comands in repository works fine and create data in table but not when i use controller

**Comment by eadwinCode** [comment]
@radthenone can you explain the problem with code examples?

**Reply by radthenone** [reply]
ok you can close this, i make a mistake with my pubsub and services, I make synchronic operactions soo why i have problem, now i added celery to my repository and all works great.

**Reply by eadwinCode** [reply]
Awesome. Good work chief
